MdePkg/Library/UefiDebugLibDebugPortProtocol/DebugLibConstructor.c

```diff
@@ -34,6 +34,7 @@ EFI_BOOT_SERVICES  *mDebugBS;
   @param  Context      Pointer to the notification function's context.
 
 **/
+STATIC // MU_CHANGE TCBZ3083 - changed to static to avoid conflicts

```

Comment:
You can leave TCBZ3083 in the commit/PR title, but please update both to indicate the function is just being made `STATIC`. That makes source history more useful.

Like: "MdePkg/UefiDebugLibDebugPortProtocol: Make ExitBootServicesCallback() static (TCBZ3083)"

Reply:
Fill in the test and integration instructions in open PRs before completion so the template text does not show up in release notes.